add docs for CI components (#110)
kubev2v/forkliftci: README.md

# forkliftci

> This repo contains collection of scripts,tools and gh actions for creating and deploying forklift cluster  and running integration e2e tests against it.

Reviewer: ```suggestion
This repository contains a collection of scripts, tools and GitHub actions for creating and deploying forklift cluster and running e2e integration tests against it.
```

* How to [install forklift on _kind_](INSTALL_FORKLIFT_ON_KIND.md).
* How to [build kubev2v-forklift and install it on _kind_](BUILD_AND_INSTALL_FORKLIFT_ON_KIND.md)

> * migration source  [providers](cluster/providers/README.md)

Reviewer: ```suggestion
* Define migration source [providers](cluster/providers/README.md)
```

> * CI e2e jobs and gh [actions](ci/README.md) 

Reviewer: ```suggestion
* Execute CI e2e jobs and GitHub [actions](ci/README.md) 
```

> * okd sno deployment ansible [playbook](cluster/okd-on-ovirt/README.md).

Reviewer: ```suggestion
* OKD single-node-OpenShift (SNO) deployment using Ansible [playbook](cluster/okd-on-ovirt/README.md).
```
